lib/strings.rego

```diff
@@ -27,7 +27,7 @@ repeat(str, x) = r {
 
 # Returns a list of lines from the provided string
 lines(str) = r {
-	r := split(str, "\n")
+	r := regex.split(`\n|\r\n`, str)
 }
```

Comment:
[nit] could be `lines(str) = regex.split(`\n|\r\n`, str)` if we wanted to keep style consistent and brief. But this is fine, too. 😃

Reply:
Right, I think we have a lot of functions to rewrite to that style (which I agree is preferable). Will fix in next PR, thanks! :)